### PR TITLE
Handle missing broken routes endpoint

### PR DESCRIPTION
**Pull request by dawehner**
When the routes endpoint fails the entire application is broken.
Let's protect against that. I'm not sure this approach is the best one, but at least its something.

### CODE REVIEW

File: packages/admin-ui/src/index.js [hunk 1]
```diff
@@ -14,10 +14,15 @@ if (process.env.NODE_ENV !== 'production' && process.env.REACT_APP_AXE) {
...
-api('admin_ui_routes').then(({ routes }) => {
-  ReactDOM.render(
-    <App serverRoutes={routes} />,
-    document.getElementById('root'),
-  );
-});
-registerServiceWorker();
+api('admin_ui_routes')
+  .catch(err => {
+    console.err(err); // eslint-disable-line no-console
```
Comment (mattgrill) [inline packages/admin-ui/src/index.js]: Would it be possible to use the error message interface to present something to the user? I don't think this is critical though.

Comment (dawehner) [reply packages/admin-ui/src/index.js]: @mattgrill I guess we could initialize the application with an error?

Comment (mattgrill) [reply packages/admin-ui/src/index.js]: Lets make a followup for that. Probably some more involved changes there.

Comment (dawehner) [reply packages/admin-ui/src/index.js]: Sure: https://github.com/jsdrupal/drupal-admin-ui/issues/600 is the followup.

**Pull request merged**